photutils/psf/psfphotometry.py

```diff
@@ -0,0 +1,282 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform PSF Photometry"""
+
+from __future__ import division
+import abc, six
```

Comment:
Indeed, it should be ``astropy.extern.six``. The logic is that we use the bundled version rather than having another runtime dependency.

Also following PEP8, it should be one package import per line